core/api/v1/keytransparency_proto/keytransparency.proto

```diff
@@ -217,7 +217,26 @@ message Epoch {
   repeated bytes log_consistency = 4;
   // log_inclusion proves that smr is part of log_root at index=srm.MapRevision.
   repeated bytes log_inclusion = 5;
-  // mutation contains mutation information.
+}
+
+// ListMutationsRequest contains the input parameters of the GetMutation APIs.
+message ListMutationsRequest {
+  // domain_id is the domain for which epochs are being requested.
```

Comment:
epochs?  not mutations?

Reply:
Thanks.  I missed this one in my API cleanup.

Reply:
Done